sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/README.md

```diff
@@ -4,7 +4,7 @@ languages:
   - java
 products:
   - azure
-  - azure-data-schemaregistry-avro
+  - azure-event-hubs
```

Comment:
Why is schema registry using event hubs tag?

Reply:
schema registry isn't a listed product in [Taxonomy Service](https://taxonomyservice.azurefd.net/TaxonomyServiceAdminPage/#/taxonomy/detail/2021-03-22T22:56:03.3422325Z!a8da30dc-3999-4cd9-9778-8380e203014c/product) so the doc generation has an error.

schema registry is a sub-service of event hubs so I use this closest parent. If I'm wrong, we can use the root "azure".

Reply:
As per offline discussion, schema registry isn't a sub service of eventhubs. Use "azure" for now until schema-registry itself is listed.

Reply:
It may not be, but that doesn't mean it's not related. These are slugs for where customers should see the sample, so if it's related, use the slug. These aren't categorizations of the sample, but a list of related products.